remix-tests version command throws error

```
% remix-tests version
internal/modules/cjs/loader.js:892
  throw err;
  ^

Error: Cannot find module 'ethjs-util'
```

Error is on path: `@remix-project/remix-tests/node_modules/@remix-project/remix-lib/src/helpers/txResultHelper.js`

--- a/libs/remix-lib/package.json
+++ b/libs/remix-lib/package.json
@@ -20,6 +20,7 @@
     "async": "^2.1.2",
     "ethereumjs-util": "^7.0.10",
     "ethers": "^4.0.40",
+    "ethjs-util": "^0.1.6",
     "events": "^3.0.0",
     "solc": "^0.7.4",
     "string-similarity": "^4.0.4",

--- a/package.json
+++ b/package.json
@@ -159,6 +159,7 @@
     "document-register-element": "1.13.1",
     "ethereumjs-util": "^7.0.10",
     "ethers": "^5.4.2",
+    "ethjs-util": "^0.1.6",
     "express-ws": "^4.0.0",
     "file-saver": "^2.0.5",
     "form-data": "^4.0.0",

